docs/source/backends.rst

```diff
@@ -192,8 +177,18 @@ As the Stepper, MPS backend does the computation on each component of the circui
 The states are represented by tensors, which are then updated at each component.
 These tensors can be seen as a big set of matrices, and the approximation is done by choosing the dimension of these matrices, called the *bond* dimension.
 
+Stepper
+^^^^^^^
+
+This simulator takes a totally different approach. Without computing the circuit's overall unitary matrix first,
+it applies the unitary matrix associated with the components in each layer of the circuit one-by-one,
+simulating the evolution of the statevector. The complexity of this backend is therefore proportional to the
+number of components. It enables simple debugging of circuits by exposing intermediate states.
+
+The `Stepper` formerly a `backend` is a `simulator` since Perceval 0.9.
+
 .. rubric:: Footnotes
 
-.. [1] Those backends technically support sampling, but to do so, they need to compute the full output distribution which
-       is totally inefficient.
+.. [1] Those backends technically support sampling, but to do so, they need to compute the full output distribution then
+       sample on it, which is totally inefficient.
```

Comment:
ça me parait important comme information, ça mériterait pas d'être en tête de page ? Sinon je mettrais les mot "then" et "inefficient" en (au choix) gras / surligné / majuscule
ET
Je remettrais en toutes lettres les algos que ça concerne genre :
Naive, Stepper and MPS backends technically support sampling, but to do so, they need to compute the full output distribution then sample on it, which is totally inefficient.